dbms/src/Flash/Mpp/Utils.cpp

```diff
@@ -85,9 +85,21 @@ std::string GenMppVersionErrorMessage(int64_t mpp_version)
 }
 
 // Get latest mpp-version supported by TiFlash
-int64_t GetMppVersion()
+MppVersion GetMppVersion()
 {
-    return (NewestMppVersion);
+    return NewestMppVersion;
 }
 
-} // namespace DB
\ No newline at end of file
+MPPDataPacketVersion GetMPPDataPacketVersion(MppVersion mpp_version)
+{
+    switch (mpp_version)
+    {
+    case MppVersion::MppVersionV0:
+        return MPPDataPacketVersion::MPPDataPacketV0;
+    case MppVersion::MppVersionV3:
+        return MPPDataPacketVersion::MPPDataPacketV2;
+    default:
```

Comment:
why make `MPPDataPacketV1` as the default value? If there is `MppVersionV4`, will it use `MPPDataPacketV1`?